docs/constants/navigation.js

```diff
@@ -447,7 +410,9 @@ function pagesFromDir(dir) {
 function pageUrl(file) {
   const filePath = path.parse(file);
   const { pathname } = new URL(path.relative(PAGES_DIR, file), 'https://docs.expo.dev');
-  return pathname.replace(filePath.base, filePath.name === 'index' ? '' : filePath.name);
+  return pathname
+    .replace(filePath.base, filePath.name === 'index' ? '' : filePath.name)
+    .replace(/\/$/, '');
```

Comment:
Internal pathnames should not add a trailing slash, even when they refer to `/eas/index.md`. This is a fix to get `/eas` as page URL from those paths.